python/generators.py.in

```diff
@@ -200,10 +200,12 @@ def single_4gev_e_upstream_tagger() :
     the field and arrive at the target at approximately [0, 0, 0] (assuming 
     it's not smeared).
 
-    The gun position below requires the particles to be fired at 4.5 degrees.
+    The gun position of [ -27.926 , 0 , -700 ] # mm requires the particles to be fired at theta = 4.5 degrees.
+    The gun position of [ -27.926 , 0 , -880 ] # mm requires the particles to be fired at theta = 5.65 degrees.
```

Comment:
please change the x coordinate to -44 in this more upstream case

Reply:
OK, fixed!